lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
```

Comment:
I mean you need to pass in analogPins if you wish to both skipCapabilities AND use analogPins :)